rundeckapp/src/main/groovy/org/rundeck/app/data/providers/GormPluginMetaDataProvider.groovy

```diff
@@ -26,7 +26,7 @@ class GormPluginMetaDataProvider implements PluginMetaDataProvider{
 
     @Override
     void deleteAllByProjectAndKeyLike(String project, String key) {
-        def found = PluginMeta.findAllByProjectAndKeyLike(project, key)
+        def found = PluginMeta.findAllByProjectAndKeyLike(project, key + '/%')
```

Comment:
this should accept the parameter "key" as the LIKE query , so should not append anything. Also I suggest renaming the parameter to `keyLike` to be clear